src/visualization/__tests__/fixtures/pathways/her2_pathway.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "UPenn Breast Cancer Pathway: Early Stage HER2+ Pathway",
-    "description": "Pathway for the UPenn Early Stage HER2+ Initial Medical Consult Breast Cancer Pathway",
+    "name": "Breast Cancer Pathway: Early Stage HER2+ Pathway",
+    "description": "Pathway for the Early Stage HER2+ Initial Medical Consult Breast Cancer Pathway",
     "library": "upenn_her2_library.cql",
```

Comment:
I don't think there's any actual impact since all the tests are passing, but could you update the `library` path for consistency in this file and the other test fixture?

Reply:
Updated here and in `src/engine/__tests__/fixtures/pathways/her2_pathway.json`